rust/cardano-chain-follower/src/chain_sync_live_chains.rs

```diff
@@ -87,6 +93,13 @@ impl ProtectedLiveChainBlockList {
             chain.lower_bound(Bound::Excluded(point))?
         };
 
+        if strict {
```

Comment:
Can remove because of the suggestion above which incorporates it where its really needed.